### ISSUE TITLE
AppImage (8.3.3) will not run

### ISSUE THREAD

**Issue by 2011** [ONLYOFFICE/appimage-desktopeditors#64]
I only get a white window (with something spinning in the center). The terminal displays the following error messages:

```
(DesktopEditors:521): dbind-WARNING **: 15:32:29.639: Couldn't connect to accessibility bus: Failed to connect to socket /root/.cache/at-spi/bus_0: Permission denied

(DesktopEditors:521): Gtk-WARNING **: 15:32:29.840: gtk_disable_setlocale() must be called before gtk_init()
MESA-INTEL: warning: Ivy Bridge Vulkan support is incomplete

(DesktopEditors:521): GLib-GIO-CRITICAL **: 15:32:44.498: g_dbus_proxy_new: assertion 'G_IS_DBUS_CONNECTION (connection)' failed
```

**Comment by uc-ai2** [comment]
Hello, @2011!
It looks like you are trying to run the .AppImage as root. Try running it as a normal user, as running it with root privileges can cause problems.

**Comment by 2011** [comment]
No, I have always attempted to run it as an ordinary user. I don't know why it tries to access /root.

**Comment by uc-ai2** [comment]
Hi @2011!
We weren’t able to reproduce the issue on any of our test Linux systems. Could you please let us know which Linux distribution and desktop environment you’re using (e.g. GNOME, KDE)? If possible, also specify the window manager in use.
Version 9.0 was recently released — does the issue still occur with this version?

**Comment by 2011** [comment]
Sorry for the delay in replying.

I have compiled the entire distribution (everything that I can) from source. I use the fluxbox window manager.

The problem still exists with version 9.0.

Somewhat oddly, I have discovered that I can (mostly) run the AppImage remotely (over ssh).

When I start it (remotely), I do not get the "Couldn't connect..." error, but instead (at the very end of the startup messages in the terminal, I get:

```
libva error: vaGetDriverNames() failed with unknown libva error
```

After the program loads, I can create a new (didn't really do anything with them - just verified that the interface loaded) Document, Presentation, or PDF. However, attempting to create a new spreadsheet produces the following screen, with sshd and editors_helper consuming about 100 percent CPU (combined) of one processor, and the program becoming unresponsive to any additional input (which requires killing the AppImage externally):

<img width="976" height="800" alt="Image" src="https://github.com/user-attachments/assets/25e741d6-ed78-4283-be48-6697636b8bb3" />

Let me know if you want me to file a separate (new) issue for the latter problem.

**Comment by 2011** [comment]
Just a brief update. I added the environment variable 'NO_AT_BRIDGE=1' when starting the AppImage, and that got rid of the "Couldn't connect to accessibility bus..." error, but the program still wouldn't run (still nothing but the white window with the circle of spinning dots in the center).

Perhaps the last message in my initial post has more relevance to the cause of the error.

**Comment by uc-ai2** [comment]
Hi @2011!
Your system is likely missing the libraries required to run the AppImage, or the kernel was compiled suboptimally. I recommend trying two solutions:
1. Install flatpak and use it to install DesktopEditors.
2. Switch to an operating system with officially declared support.

If you manage to solve the problem with your current OS, please share how you did it. This information will help developers improve AppImage compatibility with various systems.

**Comment by 2011** [comment]
> Hi [@2011](https://github.com/2011)! Your system is likely missing the libraries required to run the AppImage, or the kernel was compiled suboptimally. I recommend trying two solutions:

First, AppImages should INCLUDE all libraries necessary to run them, except for those that appear on the exclude list at:

https://github.com/AppImageCommunity/pkg2appimage/blob/master/excludelist

Second, the "missing libraries" explanation makes no sense, given that people can (except for the spreadsheet) run the app remotely (from an absolutely minimal distribution).

> 1. Install flatpak and use it to install DesktopEditors.
> 2. Switch to an operating system with officially declared support.

Neither of those suggestions appeal to me in the slightest. I have invested literally hundreds of hours setting up my system exactly the way that I want it. I actually find your suggestion to change to another operating system quite insulting.

Most AppImages (including the other major Linux office suite) run flawlessly on my machine, and I can't justify installing the entire flatpak system (which requires numerous dependencies that I do not have installed) just to run one program
